fusor-ember-cli/package.json

```diff
@@ -20,7 +20,7 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.1.2",
-    "ember-cli": "1.13.8",
+    "ember-cli": "1.13.12",
```

Comment:
Unfortunately, the hook for outputReady wasn't available in 1.13.8.  1.13.12 is the most recent ember-cli version that uses the same version of Ember we're using.  It has the additional benefit of not having incorrect generators which .8 unfortunately does.  @eriknelson has ben using .12 for awhile without issues, so perhaps this is the time to bump up.